pkg/ccip/chainaccessor/ton_accessor.go

```diff
@@ -664,21 +664,21 @@ func (a *TONAccessor) GetChainFeePriceUpdate(ctx context.Context, selectors []cc
 			continue
 		}
 		if err != nil {
-			// TODO: possibly return new map or continue
-			return nil, fmt.Errorf("failed to get chain fee price updates: %w", err)
+			a.lggr.Errorw("failed to query destinationChainGasPrice, continuing with other selectors", "selector", selector, "err", err)
+			continue
 		}
 
 		value, err := result.Cell(0)
 		if err != nil {
-			// TODO: possibly return new map or continue
-			return nil, fmt.Errorf("failed to get chain fee price updates: %w", err)
+			a.lggr.Errorw("failed to get value from result.Cell(0), continuing with other selectors", "selector", selector, "err", err)
+			continue
 		}
 
 		// HACK: we read the value as Timestamped since the binary layout is compatible, so that we match TimestampedBig (two values packed together)
 		var update feequoter.TimestampedPrice
 		if err := tlb.LoadFromCell(&update, value.BeginParse()); err != nil {
-			// TODO: possibly return new map or continue
-			return nil, fmt.Errorf("failed to get chain fee price updates: %w", err)
+			a.lggr.Errorw("failed to unmarshal TimestampedPrice, continuing with other selectors", "selector", selector, "err", err)
```

Comment:
@ogtownsend I didn't continue on these errors because they're pretty significant failures that are unlikely to recover. I'd rather fail than return potentially incorrectly parsed data